bucket_0F/pekwm/specification

```diff
@@ -0,0 +1,31 @@
+DEF[PORTVERSION]=	0.1.17
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		pekwm
+VERSION=		${PORTVERSION}
+KEYWORDS=		x11_wm
+VARIANTS=		standard
+SDESC[standard]=	Light, unobtrusive and configurable WM
+HOMEPAGE=		http://pekwm.org
+CONTACT=		Michael_Reim[kraileth@elderlinux.org]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		http://pekwm.org/projects/pekwm/files/
+DISTFILE[1]=		pekwm-${PORTVERSION}.tar.bz2:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+FPC_EQUIVALENT=		x11-wm/pekwm
+
+BUILDRUN_DEPENDS=	png:single:standard
+USES=			c++:single cclibs:single gmake iconv jpeg pkgconfig
+XORG_COMPONENTS=	xft xpm x11 sm ice xext xrandr
+MUST_CONFIGURE=		gnu
+# CONFIGURE_ARGS=		--enable-shape --enable-xinerama --enable-xrandr \
+# --enable-xft --enable-image-xpm --enable-image-jpeg \
+# --enable-image-png --disable-debug --disable-pedantic
+# For some reason splitting lines properly breakes configure...
```

Comment:
Splitting the lines somehow confuses the configure script. If uncommented and properly indented (BTW. if would be nice if lines with tabs could be commented out for testing purposes, too. Currently this is forbidden because a space is required after each comment sign and spaces before tabs are disallowed), the configure phase fails. I admit that I do not understand how this can make any difference.

Here are some lines that appear in the log when breaking up the lines:

`configure: WARNING: you should use --build, --host, --target`
`configure: WARNING: invalid host type:  --enable-xft`
`configure: WARNING: you should use --build, --host, --target`
`configure: WARNING: invalid host type:  --enable-image-png`

as well as:

`checking build system type... x86_64-raven-freebsd11`
`checking host system type... configure: error: /bin/sh ./config.sub  --enable-xft failed`
`configure: WARNING: cache variable ac_cv_host contains a newline`

Whatever option is passed after the first escaped line break seems to be mistaken for host type definition... Passing all options in just one line it works perfectly well however.